java/src/main/resources/application.yml

```diff
@@ -1,6 +1,13 @@
 spring:
+  profiles:
+    include:
+      - "@aws.profile.name@"
+      - "@azure.profile.name@"
+      - "@cosmosdb.profile.name@"
+      - "@dynamodb.profile.name@"
+
   application:
-    name: stacks-api-cqrs
+    name: com.amido.stacks-api-cqrs-events
```

Comment:
Why the app name change? The old one seems OK to me and changing it may break other stuff such as the pipeline if it depends on this.

Reply:
fixed